src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -85,24 +88,20 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### Path parameters
 
-    Sometimes you want to name certain parts of the path and access them as parameters.
-
+    When defining an explicit path, you may mark certain parts of the path, to have them available later.
+    Sections of the path that start with a `:` will generate a method that returns that param in the action.
     ```crystal
     # src/actions/users/show.cr
     class Users::Show < BrowserAction
       get "/users/:my_user_id" do
-        text "User with an id of \#{my_user_id}"
+        text "User has id \#{my_user_id}"
```

Comment:
I don't think either of these make that much sense since we didn't actually find a user. Maybe `text "user_id from path is: #{my_user_id}"`?